app/(saas)/quests/[questId]/_features/contributor-guidelines/contributor-guidelines.tsx

```
@@ -0,0 +1,64 @@
+        <AlertDescription>Your mission? Dive in, communicate, and ship awesome work.</AlertDescription>
+      </Alert>
+
+      {showMore && (
+        <>
+          {/* Before starting section */}
+          <Alert>
+            <AlertTitle>Before You Start</AlertTitle>
+            <AlertDescription>
+              <ul className="list-inside space-y-2">
+                <li>✓ Know your Quest! – Read the description, check the issues, and get the context.</li>
+                <li>✓ Set up & explore – Install what you need, scan the docs, and make sure you're ready to go.</li>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)